common/src/main/java/com/wynntils/screens/trademarket/TradeMarketSearchResultHolder.java

```diff
@@ -73,6 +82,12 @@ public void onContainerSetContent(ContainerSetContentEvent.Pre event) {
         // Reset the empty item count,
         // set content packets mean we are on a new page
         pageItemCount = 0;
+
+        if (event.getItems().size() >= SORT_MODE_SLOT) {
```

Comment:
Why is the if needed here, isn't the results screen always going to have more than 53 items?

Reply:
In theory not needed, just defensive code